### PR TITLE
Implement Codacy coverage badge

### PR DESCRIPTION
**Pull request by g105b**
I've just started using Codacy in preference to Scrutinizer. I noticed Shields had a Codacy grade indicator, but no coverage indicator, so here's the implementation.

+ The existing grade badge has been changed, but is backwards compatible (so people using the URL _without_ the `grade/` part still continue getting the correct grade.
+ New URLs for Codacy shields identify the `grade/` or `coverage/`.

Tested locally, please let me know if there are any questions.

### CODE REVIEW

**Comment by g105b** [comment]
Would be really useful to start using this on a project soon. Any chance of a merge, @espadrine ?

File: try.html [hunk 1]
```diff
@@ -666,13 +666,21 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='/wordpress/plugin/r/akismet.svg' alt=''/></td>
     <td><code>https://img.shields.io/wordpress/plugin/r/akismet.svg</code></td>
   </tr>
-  <tr><th> Codacy: </th>
-    <td><img src='/codacy/e27821fb6289410b8f58338c7e0bc686.svg' alt=''/></td>
-    <td><code>https://img.shields.io/codacy/e27821fb6289410b8f58338c7e0bc686.svg</code></td>
+  <tr><th> Codacy grade: </th>
+    <td><img src='https://img.shields.io/codacy/grade/e27821fb6289410b8f58338c7e0bc686.svg' alt=''/></td>
```
Comment (espadrine) [inline try.html]: Could you start the `src=` with a slash `/`?

File: try.html [hunk 2]
```diff
@@ -666,13 +666,21 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
...
+    <td><code>https://img.shields.io/codacy/grade/e27821fb6289410b8f58338c7e0bc686.svg</code></td>
   </tr>
-  <tr><th> Codacy branch: </th>
-    <td><img src='/codacy/e27821fb6289410b8f58338c7e0bc686/master.svg' alt=''/></td>
-    <td><code>https://img.shields.io/codacy/e27821fb6289410b8f58338c7e0bc686/master.svg</code></td>
+  <tr><th> Codacy branch grade: </th>
+    <td><img src='https://img.shields.io/codacy/grade/e27821fb6289410b8f58338c7e0bc686/master.svg' alt=''/></td>
```
Comment (espadrine) [inline try.html]: Same here.

File: try.html [hunk 3]
```diff
@@ -666,13 +666,21 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
...
+    <td><code>https://img.shields.io/codacy/grade/e27821fb6289410b8f58338c7e0bc686/master.svg</code></td>
+  </tr>
+  <tr><th> Codacy coverage: </th>
+    <td><img src='https://img.shields.io/codacy/coverage/c44df2d9c89a4809896914fd1a40bedd.svg' alt=''/></td>
```
Comment (espadrine) [inline try.html]: Same here.

File: try.html [hunk 4]
```diff
@@ -666,13 +666,21 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
...
+    <td><code>https://img.shields.io/codacy/coverage/c44df2d9c89a4809896914fd1a40bedd.svg</code></td>
+  </tr>
+  <tr><th> Codacy branch coverage: </th>
+    <td><img src='https://img.shields.io/codacy/coverage/c44df2d9c89a4809896914fd1a40bedd/master.svg' alt=''/></td>
```
Comment (espadrine) [inline try.html]: Same here.

File: README.md [hunk 5]
```diff
@@ -67,6 +67,7 @@ What kind of metadata can you convey using badges?
 ## Services using the Shields standard
 * [Badger](https://github.com/badges/badgerbadgerbadger)
 * [badges2svg](https://github.com/bfontaine/badges2svg)
+* [Codacy](https://www.codacy.com)
```
Comment (espadrine) [inline README.md]: Could you remove this line please? This list is meant for entities external to shields.io.

**Comment by espadrine** [comment]
Thanks for the contribution! I fixed the things myself to avoid you the trouble.

**Comment by g105b** [comment]
Excellent! Thanks for merging. Always a pleasure to contribute to this project. Is there anything I can do to help speed up PR acceptance?

**Pull request merged**